apps/src/lib/ui/accounts/PersonalLoginDialog.jsx

```diff
@@ -42,22 +39,16 @@ export default class PersonalLoginDialog extends React.Component {
           <p>
             <strong style={styles.dangerText}>
               {i18n.personalLoginDialog_body1({
-                numStudents: dependentStudents.length,
+                numStudents:
+                  dependentStudentsCount === dependentStudentsCountLimit
+                    ? `${dependentStudentsCount}+`
```

Comment:
There's a lot of required logic to pass `dependentStudentsCount` all the way up to this component, all in an effort to communicate that there are _over_ 100 student accounts to be deleted. If we decide this isn't necessary, it can be removed.

Reply:
Putting the `+` here vs. in the i18n strings file also means this isn't getting translated, however since it's a symbol, how would this get translated if it _were_ included in the i18n file?

Reply:
Other languages might have an alternative to writing '+'. If we don't, we can leave it to the translators to decide. It should probably be moved into the stirng.

Reply:
we could either make it "... {count} or greater students ..." or have two strings for each case and have one be "... {count} students ..." and the other be "... over {count} students"